src/output-json-ike.c

```diff
@@ -107,18 +105,24 @@ static OutputInitResult OutputIKELogInitSub(ConfNode *conf, OutputCtx *parent_ct
         return result;
     }
     ikelog_ctx->file_ctx = ajt->file_ctx;
-    ikelog_ctx->cfg = ajt->cfg;
```

Comment:
Why do we remove this line ?

Reply:
Probably because the `cfg` field was also removed from the `LogIKEFileCtx`, see previous PR. 
See https://github.com/OISF/suricata/pull/5552/commits/7c99aa30cfee4cbc94a542703e8ddaea65d7f481#diff-a67ed7262d3c2ec79597649dc8dbf9aaab3f86b51f06de1b790bff4f791f46d1L54

Reply:
Makes sense